src/SimplCommerce.WebHost/appsettings.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "ConnectionStrings": {
-    "DefaultConnection": "Server=.\\sqlexpress;Database=SimplCommerce;uid=sa;pwd=sa;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=SimplCommerce;uid=sa;pwd=sa;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
@YuriiPovkh I forgot to exclude it from my commit. I am sorry for that. We do not need to change config file.